src/components/DataTable/index.jsx

```diff
@@ -7,14 +7,27 @@ import {
   oneOf,
   oneOfType,
   object,
+  bool,
 } from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
 import Table from '@material-ui/core/Table';
 import TableBody from '@material-ui/core/TableBody';
+import TableFooter from '@material-ui/core/TableFooter';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableSortLabel from '@material-ui/core/TableSortLabel';
 import TableRow from '@material-ui/core/TableRow';
+import TablePagination from '@material-ui/core/TablePagination';
 
+@withStyles(() => ({
+  noDisplay: {
+    display: 'none',
+  },
+  tablePaginationRoot: {
+    position: 'relative',
+    left: '207%',
```

Comment:
I don't think we need this. You can look at how they do it in https://material-ui.com/demos/tables/.